prac_05/word_occurrences.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
it doesn't count the letters in the words
  "  for letter in word:  
        length = length + 1   "
something like that could help and use length to link with the words